dist/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+<script defer src="runtime.bundle.js"></script><script defer src="main.bundle.js"></script></head>
+<body>
+    <h1 class="heading">To Do List</h1>
+    <div class="container">
+        <div class="box">
+            <div class="head">
+                <p>Today's to Do</p>
+                <i class="fas fa-refresh recycle_icon" id="recycle"></i>
+            </div>
+            <div class="form-head">
+                <div class="form">
+                    <input type="text" class="addList" id="addList" placeholder="Add to your list...">
+                    <i class="fas fa-arrow-right click-icon" id="addListBtn"></i>
+                </div>   
+            </div>
+            <div class="body">
+                <ul class="checkbox-list" id="to-do-lists">
+                   
+                </ul>
+              
+            </div>   
+            <div class="footer">
+                <button class="clear-list" id="clear-list">Clear All Completed</button>
+            </div>  
+        </div>
+    </div>
```

Comment:
kindly add a section tag for looking at professional codes with HTML best practice 🎉👍